crates/ironrdp-pdu/src/lib.rs

```diff
@@ -327,6 +327,17 @@ pub trait PduHint: Send + Sync + core::fmt::Debug + 'static {
     fn find_size(&self, bytes: &[u8]) -> PduResult<Option<usize>>;
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct AnyHint;
```

Comment:
I feel "any" is a bit too "wide". Maybe this should be named `RdpPduHint`? X224 and FastPath are not the kind of PDUs we may receive, especially during the CredSSP procedure